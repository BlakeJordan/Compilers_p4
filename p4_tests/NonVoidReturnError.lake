void b() {
    return 4;
}