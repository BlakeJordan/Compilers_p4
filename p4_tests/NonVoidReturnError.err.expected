2,12: Return with a value in void function
Type checking failed